controller/environment_space_scope_blackbox_test.go

```diff
@@ -0,0 +1,199 @@
+package controller_test
+
+import (
+	"context"
+	"crypto/rsa"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/fabric8-services/fabric8-common/service"
+	testauth "github.com/fabric8-services/fabric8-common/test/auth"
+	testsuite "github.com/fabric8-services/fabric8-common/test/suite"
+	"github.com/fabric8-services/fabric8-common/token"
+	"github.com/fabric8-services/fabric8-env/app"
+	"github.com/fabric8-services/fabric8-env/app/test"
+	"github.com/fabric8-services/fabric8-env/configuration"
+	"github.com/fabric8-services/fabric8-env/controller"
+	"github.com/fabric8-services/fabric8-env/gormapp"
+	"github.com/goadesign/goa"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+/*
+Setup:
```

Comment:
could you add a comment to explain that the users' scopes are provided by the `handleSpaceScopeRequest` function?

Reply:
just want to keep test_definition here.  also not very big test and only one handler there.